src/components/pages/pageNotFound/pageNotFound.js

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+import React from 'react';
+
+import './pageNotFound.css';
+
+export const PageNotFound = () => (
```

Comment:
This doesn't need to be a component?

Reply:
It is. React components can be either class components or functional components. https://reactjs.org/docs/components-and-props.html#functional-and-class-components